dqc_us_rules/dqc_us_0046.py

```diff
@@ -33,10 +33,6 @@ def _find_errors(val):
     calc_children = _load_config(_CONFIG_JSON_FILE)
     if not calc_children:
         return  # nothing can be checked
-    # convert children lists into sets for faster "in" function processing
-    calc_children = dict(
-        (key, set(value)) for key, value in calc_children.items()
-    )
```

Comment:
After talking with Allen, he found that making the check against a list is actually faster than setting the list and then making the check. So the above code removes an unneeded step.
👍